server/controllers/products.js

```diff
@@ -55,15 +55,5 @@ module.exports = {
         console.error('Unable to retrieve Item data: ', err);
         res.sendStatus(404);
       });
-  },
-
-  getStyles: (req, res) => {
-    const productId = req.params.product_id;
-    axios.get(`${ATELIER_API}/products/${productId}/styles`, { headers: { authorization: API_TOKEN } })
-      .then(({ data }) => res.status(200).json(data))
-      .catch((err) => {
-        console.error('Unable to retrieve item style: ', err);
-        res.sendStatus(404);
-      });
-  },
-};
```

Comment:
I think you need to communicate with @beanjermin about which `getStyles()` to use, because I think you're both deleting each other's methods.

Reply:
Refer to 
https://github.com/Coffea-Exotica-Enticers/FEC/pull/26/files/d53724e0c9f4519f1ca2dd8176a135162b3decbf#diff-023c271cb411b642a45481bd08d1283ba562c026e1fa5954fe527d7ece0dd404

Reply:
yeah I asked him now